ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java

```diff
@@ -115,9 +115,17 @@ public abstract class IgniteAbstractClient extends DB {
 
   /**
    * Used to choose storage engine (e.g., 'aipersist' or 'rocksdb').
+   * @deprecated Removed in <a href="https://ggsystems.atlassian.net/browse/IGN-23905">IGN-23905</a>
    */
+  @Deprecated
   protected static String dbEngine;
 
+  /**
+   * Used to choose storage profiles separated by comma
+   * (e.g., 'default', 'default_aimem', 'default_aipersist,default_rocksdb', etc.).
+   */
+  protected static String storageProfiles;
```

Comment:
Why not to use List<String> for the config parameters that is a list string by its nature?

Reply:
It's just a string we grab from property and send unchanged to the expression `STORAGE_PROFILES='%s'`